### PR TITLE
add ref to mounting emptyDir for ephemeral storage

### PR DESCRIPTION
**Pull request by wendtek**

### CODE REVIEW

**Comment by stuart23** [comment]
Can we also add an example in this doc https://github.com/astronomer/docs/blob/main/astro/kubernetespodoperator.md showing the settings needed for KPO to mount this vol?

**Review by jwitz — COMMENTED** [review]
Thanks for getting this started @wendtek ! I add some suggestions as well as some documentation for the KPO. Can you please: 

- Review the content I added to make sure it aligns with your note in the resource reference guide
- Provide an example of how a standard task would write to `/ephemeral`

Thank you!

File: astro/kubernetespodoperator.md [hunk 1]
```diff
@@ -98,7 +98,47 @@ KubernetesPodOperator(
...
+    name="<pod-name>",
+    resources=compute_resources,
+    task_id="<task-name>",
+    get_logs=True,
+    volume_mounts=volume_mounts,
+    volumes=[volume],
+    )
+    ```
+    
+    Your task can now utilize the configured volume as storage for simple operations. However, Astronomer recommends setting Airflow connection to an external storage service such as Amazon S3 for larger operations requiring storage. See [Passing data between Airflow tasks](https://www.docs.astronomer.io/learn/airflow-passing-data-between-tasks/).
```
Comment (jwitz) [inline astro/kubernetespodoperator.md:139]:
```suggestion
    Your task can now write data to the configured volume. Astronomer recommends using this volume only with small amounts of data. For larger operations requiring storage, Astronomer recommends setting an Airflow connection to an external storage service such as Amazon S3 . See [Passing data between Airflow tasks](https://www.docs.astronomer.io/learn/airflow-passing-data-between-tasks/).
```

File: astro/kubernetespodoperator.md [hunk 2]
```diff
@@ -98,7 +98,47 @@ KubernetesPodOperator(
 
 Applying the code above ensures that when this DAG runs, it will launch a Kubernetes Pod with exactly 800m of CPU and 3Gi of memory as long as that infrastructure is available in your cluster. Once the task finishes, the Pod will terminate gracefully.
 
-## Run images from a Private Registry
+### Mount a temporary directory (_AWS only_)
+
+Astronomer provisions `m5d` workers with NVMe SSD volumes. These volumes can be used by tasks for storage which persists as long as the task is running. To run a task run the KubernetesPodOperator that utilizes temporary storage:
```
Comment (stuart23) [inline astro/kubernetespodoperator.md:103]: Or m6id https://docs.astronomer.io/astro/resource-reference-aws#worker-node-types

File: astro/kubernetespodoperator.md [hunk 3]
```diff
@@ -98,7 +98,47 @@ KubernetesPodOperator(
...
+    name="<pod-name>",
+    resources=compute_resources,
+    task_id="<task-name>",
+    get_logs=True,
+    volume_mounts=volume_mounts,
+    volumes=[volume],
+    )
+    ```
+    
+    Your task can now write data to the configured volume. Astronomer recommends using this volume only with small amounts of data. For larger operations requiring storage, Astronomer recommends setting an Airflow connection to an external storage service such as Amazon S3 . See [Passing data between Airflow tasks](https://www.docs.astronomer.io/learn/airflow-passing-data-between-tasks/).
```
Comment (stuart23) [inline astro/kubernetespodoperator.md:139]: Can you point to the resources table telling them how much space is available? "small amounts of data" is too vague.

Comment (stuart23) [inline astro/kubernetespodoperator.md:139]:
```suggestion
    Your task can now write data to the configured volume. Astronomer recommends using this volume only with small amounts of data. For larger operations requiring storage, Astronomer recommends setting an Airflow connection to an external storage service such as Amazon S3, see [Passing data between Airflow tasks](https://www.docs.astronomer.io/learn/airflow-passing-data-between-tasks/).
```

**Review by stuart23 — CHANGES_REQUESTED** [review]

File: astro/kubernetespodoperator.md (hunk 3 above)
Comment (stuart23) [reply astro/kubernetespodoperator.md]: There is 20GB capacity for all instance types except for the m5d and m6id types - we need to tell them how much space is available according to https://aws.amazon.com/ec2/instance-types/m6i/ & https://aws.amazon.com/ec2/instance-types/m5/

Comment (stuart23) [reply astro/kubernetespodoperator.md]: We don't necessarily need to give them numbers - we can just link the above

File: astro/kubernetespodoperator.md [hunk 4]
```diff
@@ -98,7 +98,49 @@ KubernetesPodOperator(
...
+    name="<pod-name>",
+    resources=compute_resources,
+    task_id="<task-name>",
+    get_logs=True,
+    volume_mounts=volume_mounts,
+    volumes=[volume],
+    )
+    ```
+    
+    Your task can now write data to the configured volume. Astronomer recommends that your tasks use no more storage than 20GB divided by your configured maximum tasks per worker. See [Worker autoscaling logic](configure-worker-queues.md#worker-autoscaling-logic). For larger operations requiring storage, Astronomer recommends setting an Airflow connection to an external storage service such as Amazon S3. See [Passing data between Airflow tasks](https://www.docs.astronomer.io/learn/airflow-passing-data-between-tasks/).
```
Comment (wendtek) [inline astro/kubernetespodoperator.md:141]: I don't think we should be recommending 20GB here unless we have language excepting the NVME based instances. Possibly better to just link to the AWS docs about instance storage.

File: astro/kubernetespodoperator.md [hunk 5]
```diff
@@ -98,7 +98,49 @@ KubernetesPodOperator(
...
+    name="<pod-name>",
+    resources=compute_resources,
+    task_id="<task-name>",
+    get_logs=True,
+    volume_mounts=volume_mounts,
+    volumes=[volume],
+    )
+    ```
+    
+    Your task can now write data to the configured volume.
```
Comment (jwitz) [inline astro/kubernetespodoperator.md:141]:
```suggestion
```

**Comment by stuart23** [comment]
Can we merge https://github.com/astronomer/docs/pull/1454 into this and then merge them both to main? I am getting confused with the differences between them

**Pull request merged**